code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -246,6 +246,16 @@
 	description = "A salt made of sodium chloride. Commonly used to season food."
 	reagent_state = SOLID
 	color = "#FFFFFF" // rgb: 255,255,255
+	overdose_threshold = 30
+	
+/datum/reagent/consumable/sodiumchloride/overdose_start(mob/living/M)
+	M << "<span class='userdanger'>You're too salty!</span>"
+
+
+/datum/reagent/consumable/sodiumchloride/overdose_process(mob/living/M)
+	M.adjustToxLoss(1.5*REM, 0)
+	M.adjustFireLoss(1*REM, 0)
```

Comment:
yes but like at 30u assuming this processes 30 times that is literally like taking two hits from a welder to the face because you ate too much salt
thats retarded